src/masonite/commands/MakeModelCommand.py

```diff
@@ -0,0 +1,54 @@
+"""New Model Command."""
```

Comment:
I though we would just make a new parameter inside the __init__ and then in Masonite we pass in the default model directory?